### DISCUSSION TITLE
Combobox Figma Component: Resize item list

### DISCUSSION THREAD

**Discussion by kkenna3** [twilio-labs/paste#2652] [Help: Figma] [answered]
Hi, I was using the latest version of the combobox in Figma and noticed that if you reduce the number of list items ("open" state) and change the **Listbox** component to "Hug" contents vertically, it resizes to the bottom and not the top.

I've made a branch of the library to test out a fix by constraining the Listbox to the top instead of the bottom. [Here's the branch file](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/branch/LVSfa7d4YKkJ8trTtJ28st/Paste-Components?node-id=13068%3A71532) (not everything has been tested).

<img width="742" alt="Screen Shot 2022-08-30 at 10 01 42 AM" src="https://user-images.githubusercontent.com/102549494/187457418-b9318e70-fd82-442b-81dc-b7a00c6148ab.png">

Unfortunately for now I'll have to detach the component to get it to work, let me know if there's another fix. Thanks!

**Comment by sandsng** [comment]
Hi Kara! 

Thanks for bringing this to our attention. I've created [this JIRA ticket](https://issues.corp.twilio.com/browse/DSYS-3526) to implement a fix. I will update this thread once a fix has been merged in.

**Comment by loreina** [comment]
Hi @kkenna3, just wanted to follow up here as I'm looking at your branch. Could you share more context into why you want to have 4 options in the listbox? We generally recommend using [Checkbox Group](https://paste.twilio.design/components/checkbox#checkbox-and-checkbox-group) if there's less than 6 options for a user to select from.

**Reply by kkenna3** [reply]
Hi, in this particular instance the number of items in the dropdown is variable and dependent on what the user's created in another section of Console (there's not a limit to how many they can create). Is there another pattern that should be used for a situation like this?

**Comment by loreina** [comment, marked as answer]
Hi @kkenna3, thanks for your patience! A fix has been implemented so if the listbox height is set to "hug contents" and there are <6 options, it'll shrink to that height.
